sdk/core/System.ServiceModel.Rest/api/System.ServiceModel.Rest.net6.0.cs

```diff
@@ -137,13 +141,13 @@ public partial class HttpPipelineMessageTransport : System.ServiceModel.Rest.Cor
     {
         public HttpPipelineMessageTransport() { }
         public HttpPipelineMessageTransport(System.Net.Http.HttpClient client) { }
-        public override System.ServiceModel.Rest.Core.PipelineMessage CreateMessage() { throw null; }
+        public override System.ServiceModel.Rest.Core.PipelineMessage CreateMessage(System.ServiceModel.Rest.InvocationOptions options) { throw null; }
```

Comment:
This will end up being RequestOptions, but keeping it called InvocationOptions for now still